flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaSinkITCase.java

```diff
@@ -158,7 +167,7 @@ public void setUp() throws ExecutionException, InterruptedException, TimeoutExce
         createTestTopic(topic, 1, TOPIC_REPLICATION_FACTOR);
     }
 
-    @After
+    @AfterEach
     public void tearDown() throws ExecutionException, InterruptedException, TimeoutException {
```

Comment:
```suggestion
    void tearDown() throws ExecutionException, InterruptedException, TimeoutException {
```
could be package private